rust/toolchain.bzl

```diff
@@ -2,9 +2,6 @@
 The rust_toolchain rule definition and implementation.
 """
 
-def _get_files(labels):
-    return [f for l in labels for f in getattr(l, "files", [])]
```

Comment:
Not clear why this needed `--incompatible_depset_is_not_iterable=false` to work, but I refactored it out.